entrypoint.rb

```diff
@@ -12,6 +12,7 @@
   config.repo = ENV["GITHUB_REPOSITORY"]
   config.wait = wait.to_i
   config.workflow_name = workflow_name
+  config.allowed_conclusions = allowed_conclusions&.split(/\s*,\s*/)
```

Comment:
I originally just did `allowed_conclusions.split(/\s*,\s*/)`, but the test for entrypoint failed, I guess because this param isn't passed in in the test:

```
  1) entrypoint calls the GithubChecksVerifier service
     Failure/Error: require_relative "../../entrypoint.rb"
     
     NoMethodError:
       undefined method `allowed_conclusions?' for main:Object
     # /entrypoint.rb:15:in `block in <top (required)>'
     # /usr/local/bundle/gems/activesupport-6.1.1/lib/active_support/configurable.rb:38:in `configure'
     # /entrypoint.rb:7:in `<top (required)>'
     # ./spec/entrypoint_spec.rb:6:in `require_relative'
     # ./spec/entrypoint_spec.rb:6:in `block (2 levels) in <top (required)>'
```

Adding the `&` of course resolved this error in the test, but I'm not sure if it's just "hiding" an issue?

Reply:
The `&` operand there is the "safe navigation operator" and it's a double-edged sword. You can read more about it [here](https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)
Please use `allowed_conclusions.split(',').map(&:strip)`
We are using a default value from the action.yml, which makes the safe navigation operator useless here. 
The `strip` method removes blanks around the strings, in case users send something like "skipped, success" instead of "skipped,success" (not the space between the comma and the next word).

Reply:
I don't understand the `undefined method allowed_conclusions? for main:Object` error. Such method is not being called anywhere 🤷‍♂️

Reply:
To avoid the entrypoint_spec failing, add `allow(GithubChecksVerifier).to receive(:configure)` before requiring the entrypoint.rb file

Reply:
Sorry, I missed these replies earlier 🤦‍♂️ 😝 

Adding the `allow(GithubChecksVerifier).to receive(:configure)` seems to have solved the issue! I did understand what the safe navigation operator was doing, I was just worried it was "ignoring" a case where Nil was passed in where we didn't expect it. That should be solved now anyway.

Reply:
Yeah... since we have a default value set in the action.yml, it should not receive nil values